packages/core/src/sso/SamlConnector/index.ts

```diff
@@ -149,16 +154,15 @@ class SamlConnector {
 
       // eslint-disable-next-line new-cap
       const identityProvider = saml.IdentityProvider({
-        wantAuthnRequestsSigned: true, // Sign auth request by default
         metadata: idpMetadataXml,
       });
+
       // eslint-disable-next-line new-cap
       const serviceProvider = saml.ServiceProvider({
-        entityID,
+        entityID, // FIXME: @darcyYe
```

Comment:
@darcyYe also here. Better double check the definition for the ServiceProvider factories